src/client/public-openfin/rt-fx.json

```diff
@@ -67,7 +67,7 @@
           "reloadIgnoringCache": true,
           "zoom": true
         },
-        "url": "<BASE_URL>/openfin-window-frame",
+        "url": "<BASE_URL>/openfin-window-frame?app=FX",
```

Comment:
I might be out of date, but we used to use a window.options attribute for things like this
It is not critical, but just floating it as a more "OpenFinny" way of passing context to an app/window

Reply:
for ref, this is what I was thinking of
https://developer.openfin.co/docs/javascript/stable/Window.html#~options
..
specifically the `customData` or perhaps `customContext` attributes